src/main/game/Computer.java

```diff
@@ -0,0 +1,17 @@
+package main.game;
+
+import java.util.List;
+
+import static main.game.PlayerTypes.PLAYER_TWO_MARK;
+
+public class Computer implements Player {
+    public void playMove(Board board) {
+        String playerMove = move(board);
```

Comment:
I don't think it's necessary to save this in a variable because you only use it once. Maybe improve the naming of `move()` and then you don't need to save it in this temporary variable